.github/workflows/deploy-to-dev-manual.yml

```diff
@@ -37,16 +37,22 @@ on:
         - "chart_value"
         - "true"
         - "false"
+      database_deletion:
```

Comment:
I think this can also be called recreate_databases to keep it consistent with the other variables

Reply:
changed to "database_recreation"